frontend/src/stores/charts.js

```diff
@@ -16,6 +16,8 @@ export const useChartsStore = defineStore('charts', () => {
   const showChart = ref(false)
   const hasNodeData = ref(false)
   const chartTab = ref('timeseries')
+  const plotStyle = ref('Scatter')
```

Comment:
why do you add `plotStyle` back in here? I think this is maybe an error when resolving merge conflicts? Or was this intentional?

Reply:
I think this was just a mistake when resolving merge conflicts.